### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by blasten**
Preview in the new catalog: https://beta.webcomponents.org/preview/PolymerElements/app-layout/f2136fd52033d3b5aff51633b7e6b5e1871b2064

### CODE REVIEW

**Comment by frankiefu** [comment]
Looks good.  Can we also have links to the live demos of the Templates and Patterns, maybe have Catalog serves the live demos?

**Comment by blasten** [comment]
@frankiefu I added link to the demos to the readme.md. However, each template has its own bower.json and file structure. For now, I just pointed the demos to github.io

File: README.md [hunk 1]
```diff
@@ -93,14 +109,50 @@ A set of layout elements for your app. It includes:
...
+- **Scroll effects: Test drive**
+([Demo](https://polymerelements.github.io/app-layout/templates/test-drive) - [Source](/templates/test-drive))
+
+### Patterns
+
+Sample code for various UI patterns:
+
+- **Transform navigation:**
+As more screen space is available, side navigation can transform into tabs.
+([Demo](https://polymerelements.github.io/app-layout/patterns/transform-navigation/x-app.html) - [Source](/patterns/transform-navigation/x-app.html))
```
Comment (frankiefu) [inline README.md]: `/patterns/transform-navigation/x-app.html` -> `/patterns/transform-navigation/index.html`

Comment (blasten) [inline README.md]: Good catch. should be fixed now

**Comment by frankiefu** [comment]
LGTM

**Pull request merged**